src/routes/Overview.svelte

```diff
@@ -21,30 +21,34 @@
 <Header>Overview</Header>
 
 <p>
-  This tool helps evaluators build Accessibility Conformance Reports in the OpenACR format.
-  It guides you through the OpenACR requirements, lets you record your software's current
-  conformance for each requirement, and generates a report.
+  This web editor helps evaluators build Accessibility Conformance Reports in the <a href="https://github.com/gsa/openacr">OpenACR format</a>.
+  It is designed to help accessibility subject matter experts create machine-readable OpenACR documents. Authors will be guided in creating an
+  accessible report for the digital product or service that they are documenting. Conformance for each requirement can be documented as required
+  to generate a Section 508 report.
+</p>
+<p>
+  <a on:click="{startNew}">Start new report</a> or open an existing one.
 </p>
 
 <p>Tips for using this tool:</p>
 
 <ul>
   <li>
-    As you use the tool, the
-    <Link to="report">Report</Link>
-    page lists the success criteria that you have checked and not checked.
+    This editor saves the information you enter locally in your browser (not on our servers). To save or share your OpenACR report, you will
+    need to save it to your computer. You will need the YAML file to submit your OpenACR and to edit this report in the future.
```

Comment:
Instead of:
`You will need the YAML file to submit your OpenACR and to edit this report in the future.`

What if we changed it to:
`You will need this YAML file to continue your in-progress report, submit your OpenACR, or to update this report in the future.`